examples/external_stores.py

```diff
@@ -26,9 +26,20 @@
     }
 }
 
+# variable declaration
+management_ip = '10.1.188.204'
```

Comment:
Please follow this convention that has been set forth for user input
```
    "ip": "<ovc_ip>",
    "credentials": {
        "username": "<username>",
        "password": "<password>"
```

Reply:
Updated as per the format